scripts/publish_gh_release_notes.py

```diff
@@ -0,0 +1,86 @@
+"""
+Script used to publish GitHub release notes extracted from CHANGELOG.rst.
+
+This script is meant to be executed after a successful deployment in Travis.
+
+Uses the following environment variables:
+
+* GIT_TAG: the name of the tag of the current commit.
+* GH_TOKEN: a personal access token with 'repo' permissions. It should be encrypted using:
+
+    $travis encrypt GH_TOKEN=<token> -r pytest-dev/pytest
+
+  And the contents pasted in the ``env.global.secure`` section in the ``travis.yml`` file.
```

Comment:
> env.global.secure

Needs to be adjusted then.. having an explicit name could just have that name in the comment above then.